src/Examples/Flowsheets/power_generation/rsofc/cpu_surrogate_methods.py

```diff
@@ -0,0 +1,216 @@
+##############################################################################
+# The development of this flowsheet/code is funded by the ARPA-E DIFFERENTIATE
+# project: “Machine Learning for Natural Gas to Electric Power System Design”
+# Project number: DE-FOA-0002107-1625.
+# This project is a collaborative effort between the Pacific Northwest National
+# Laboratory, the National Energy Technology Laboratory, and the University of
+# Washington to design NGFC systems with high efficiencies and low CO2
+# emissions.
+##############################################################################
+
+
+class compressor_power_fun:
```

Comment:
As mentioned earlier, this would be an excellent demonstration of the standard surrogate modelling interface - it would be really nice if it could be updated to use it.